### PR TITLE
`max_line_length` support for Black

### PR DESCRIPTION
**Pull request by quarthex**
### What does this implement/fix? Explain your changes.

Support of the `max_line_length` option for the Black beautifier (Python).

### Does this close any currently open issues?

No.

### Any other comments?

No.

### Checklist

Check all those that are applicable and complete.

- [ ] Merged with latest `master` branch
- [ ] Regenerate documentation with `npm run docs`
- [ ] Add change details to `CHANGELOG.md` under "Next" section
- [ ] Added examples for testing to [examples/ directory](examples/)
- [ ] Travis CI passes (Mac support)
- [ ] AppVeyor passes (Windows support)

### CODE REVIEW

File: src/options.json [hunk 1]
```diff
@@ -6476,6 +6476,7 @@
         "title": "Max line length",
         "beautifiers": [
           "autopep8",
+          "black",
```
Comment (Glavin001) [inline src/options.json:6479]: Please complete the following:

- [x] Option added to Python language: https://github.com/Glavin001/atom-beautify/blob/master/src/languages/python.coffee#L23
- [ ] Option added to `Black` beautifier: https://github.com/Glavin001/atom-beautify/blob/master/src/beautifiers/black.coffee#L29

See https://github.com/Glavin001/atom-beautify/blob/master/src/beautifiers/js-beautify.coffee#L18-L25 for example:

```diff
-     Python: false
+    Python: 
+      wrap_line_length: true
```

- [ ] Re-generate the docs: https://github.com/Glavin001/atom-beautify/blob/master/docs/add-languages-and-beautifiers.md#options-documentation

```bash
npm run docs
```

File: src/beautifiers/black.coffee [hunk 2]
```diff
@@ -32,7 +32,10 @@ module.exports = class Black extends Beautifier
   beautify: (text, language, options, context) ->
     cwd = context.filePath and path.dirname context.filePath
     # `-` as filename reads from stdin
-    @exe("black").run(["-"], {
+    @exe("black").run([
+      ["-l", "#{options.max_line_length}"] if options.max_line_length?
```
Comment (Glavin001) [inline src/beautifiers/black.coffee]: I prefer using the long options so it is more readable in our code:

```suggestion
      ["--line-length", "#{options.max_line_length}"] if options.max_line_length?
```

**Comment by Glavin001** [comment]
Thank you for contributing! 🎉 Please let me know if you have any questions or need assistance with this Pull Request. 😃

File: src/options.json (hunk 1 above)
Comment (quarthex) [reply src/options.json:6479]: Should be done in 52301ec.

**Comment by Glavin001** [comment]
@stevenzeck Any feedback on your end?

**Comment by quarthex** [comment]
No more activity on this project since more than one year, so closing this pull request.

**Pull request closed without merging**